.github/scripts/check-updater.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+#
+set -e
+# shellcheck source=.github/scripts/functions.sh
+. "$(dirname "$0")/functions.sh"
```

Comment:
Do double quotes inside double quotes work? Doesn't this need escaping?

Reply:
There’s a layer between the two sets of double quotes because of how the shell parsing works.

Effectively, anything inside a `$()` is parsed as regular shell code in a context completely isolated from the surrounding code (albeit inheriting the environment variables from the surrounding context), so something like `"$(echo ")` would fail to parse due to the unmatched double quote inside the `$()`, not due to the unmatched parenthesis.